geniza/corpus/templates/corpus/snippets/document_result.html

```diff
@@ -107,15 +107,15 @@ <h3 class="sr-only">{% translate 'Tags' %}</h3>
                 {# list of tuples of (IIIF image, label) #}
                 {% for image in document.iiif_images|slice:":3" %}
                     <li class="image-{{ forloop.counter }}">
-                        <a href="{% url 'corpus:document' document.pgpid %}">
+                        <a href="{% url 'corpus:document' document.pgpid %}" data-turbo-frame="main" target="_top">
                             <img src="{{ image.0|iiif_image:"size:width=250" }}" loading="lazy" alt="{{ image.1 }}">
                         </a>
                     </li>
                 {% endfor %}
             </ul>
         {% endif %}
         {# view link #}
-        <a class="view-link" href="{% url 'corpus:document' document.pgpid %}">
+        <a class="view-link" href="{% url 'corpus:document' document.pgpid %}" data-turbo-frame="main" target="_top">
```

Comment:
I don't think we need the `target="_top"` — I think turbo action set to advance and turbo frame `main` is sufficient; with `_top` I get a flash of unstyled pages; without it I think everything is working properly.

Reply:
Quick note on this: the reason why I added `target="_top"` was because I kept getting scrolled most of the way down the document detail page when I navigated with this link. Turbo specifically says in the docs that links should always scroll back to the top unless a hash/anchor is in the URL, so I couldn't figure out why this was happening… the `target="_top"` was a quick fix. I should have mentioned earlier!

Reply:
I started coding a fix for this where we do `window.scrollTo(0, 0)`—but when you navigate back to the search results, you end up losing your place in the search results. Probably preferable to being scrolled down when you click on the document, but not sure!

Here's the documentation I mentioned: https://turbo.hotwired.dev/handbook/drive#application-visits
> If the visit’s location includes an anchor, Turbo Drive will attempt to scroll to the anchored element. Otherwise, it will scroll to the top of the page.